WebdevPeriod3/wwwroot/js/subsystemlist.js

```diff
@@ -82,7 +82,8 @@ function resultsToHTML(results) {
     return results.map(result => `<li data-id="${result.id}" class="bg-gray-100 p-3 mt-1 cursor-pointer rounded-sm hover:bg-gray-200" data-name="${result.name}">${result.name} - ${result.description}</li>`).join("\n");
 }
 
-function createSubSystemStore() {
+function createSubSystemStore(initialSubSystems) {
+    console.log(initialSubSystems);
```

Comment:
Debugging leftover?

Reply:
Only the second line is a leftover, I did intentionally add a new parameter to the function.